source/user-manual/reference/ossec-conf/rule_test.rst

```diff
@@ -0,0 +1,85 @@
+.. Copyright (C) 2020 Wazuh, Inc.
+
+.. _reference_ossec_rule_test:
+
+rule_test
+=========
+
+.. topic:: XML section name
+
+	.. code-block:: xml
+
+		<rule_test>
+		</rule_test>
+
+Here is how to configure the wazuh-logtest module. It allows to test rules and decoders from Wazuh API and Kibana through Wazuh app 
+
+Options
+-------
+
+- `enabled`_
+- `threads`_
+- `max_sessions`_
+- `session_timeout`_
+
+
+
+enabled
+^^^^^^^
+
+Enables the module.
+
++--------------------+------------------------+
+| **Default value**  | yes                    |
++--------------------+------------------------+
+| **Allowed values** | yes/no                 |
++--------------------+------------------------+
+
+threads
+^^^^^^^
+
+Number of wazuh-logtest threads.
+
++--------------------+---------------------------------+
+| **Default value**  | 1                               |
++--------------------+---------------------------------+
+|                    | a number between 1 and 128,     |
+| **Allowed values** | or `auto` to create one thread  |
+|                    | per CPU                         |
++--------------------+---------------------------------+
+
+max_sessions
+^^^^^^^^^^^^
+
+Max number of users connected simultaneously.
+
++--------------------+---------------------------------+
+| **Default value**  | 1                               |
++--------------------+---------------------------------+
+| **Allowed values** | A number between 1 and 500      |
++--------------------+---------------------------------+
+
+session_timeout
+^^^^^^^^^^^^^^^
+
+Time required to delete a session and its resources after the last user interaction.
+
++--------------------+---------------------------------------------+
+| **Default value**  | 15m                                         |
++--------------------+---------------------------------------------+
+|                    | A number to represent seconds, to represent |
```

Comment:
Be more clear about seconds/minute/hour syntax

Reply:
Change it to` A positive number that should contain a suffix character indicating a time unit, such as, s (seconds), m (minutes), h (hours)`
View string_op.c:wlogtest and LOGTEST_LIMIT_SESSION_TIMEOUT, 
The max value is 365 days